### ISSUE TITLE
Unable to save schedule

### ISSUE THREAD

**Issue by TMRE** [basildane/WakeOnLAN#185]
I tried a variety to different schedules, I keep getting the same error msg.

![error](https://user-images.githubusercontent.com/78712812/121742058-935dea00-cabc-11eb-9c25-e1c94c5f9323.PNG)

**Comment by basildane** [comment]
Yes, unfortunately that is a bug in the Microsoft scheduler interop api.  I reported it to them in 2014.  They can't even tell me what the error means, and there is no documentation for it at all.
It affects some people and not others.  No one knows why.

One thing you can do is create scheduled tasks directly in windows instead of WOL, because WOL is just putting tasks into the regular windows task scheduler.  There is help on this.  (use the command line commands).

If you get stuck I can help you.
